dbms/src/Server/Server.cpp

```diff
@@ -569,6 +571,22 @@ int Server::main(const std::vector<std::string> & args)
         for (auto & server : servers)
             server->start();
 
+        {
+            std::string report;
+            auto memory_amount = getMemoryAmount();
+            if (memory_amount)
+               report += " RAM = " + std::to_string(memory_amount/1024/1024) + "MB;";
+            auto cores = getNumberOfPhysicalCPUCores();
+            if (cores)
```

Comment:
if (data.num_logical_cpus == 0)                
    return std::thread::hardware_concurrency();

std::thread::hardware_concurrency() :
number of concurrent threads supported. If the value is not well defined or not computable, returns ​0​.

So it can, on hardware from UFO